_posts/2022-10-19-arrow-flight-sql-jdbc.md

```diff
@@ -0,0 +1,115 @@
+
+Flight SQL is a protocol[^1] for client-server database interactions.
+It defines how a client should talk to a server and execute queries,
+fetch result sets, and so on.  Underneath, it builds on [Arrow Flight
+RPC][flight], a framework for efficient transfer of Arrow data across
+the network.  While Flight RPC is flexible and can be used in any type
+of application, from the beginning, it was designed with an eye
+towards the kinds of use cases that Flight SQL supports.
+
+With this new JDBC driver, applications can talk to any database
+server implementing the Flight SQL protocol using familiar JDBC APIs.
+Underneath, the driver sends queries to the server via Flight SQL and
+adapts the Arrow result set to the JDBC interface, so that the
+database can support JDBC users without implementing additional APIs
+or its own JDBC driver.
+
+[^1]: Despite the name, Flight SQL is **not** a SQL dialect, nor is it
+    even tied to SQL itself!
+
+## Why use JDBC with Flight SQL?
+
+JDBC offers a row-oriented API, which is opposite of Arrow's columnar
+structure.  However, it is a popular and time-tested choice for many
+applications.  For example, many business intelligence (BI) tools take
+advantage of JDBC to interoperate generically with multiple databases.
+An Arrow-native database may still wish to be accessible to all of
+this existing software, without having to implement multiple client
+drivers itself.  Additionally, columnar data transfer alone can be a
+[significant speedup][hannes] for analytical use cases.
+
+This JDBC driver implementation demonstrates the generality of Arrow
+and Flight SQL, and increases the reach of Arrow-based applications.
+Additionally, an [ODBC driver implementation][dremio-odbc] based on
+Flight SQL is also available courtesy of Dremio, though it is not yet
+part of the Arrow project due to dependency licensing issues.
+
+Now, a database can support the vast body of existing code that uses
+JDBC or ODBC, as well as Arrow-native applications, just by
+implementing a single wire protocol: Flight SQL.  Some projects
+instead do things like reimplementing the Postgres wire protocol to
+benefit from its existing drivers.  But for Arrow-native databases,
+this gives up the benefits of columnar data.  On the other hand,
+Flight SQL is:
+
+1. Columnar and Arrow-native, using Arrow for result sets to avoid
+   unnecessary data copies and transformations;
+2. Designed for implementation by multiple databases, with high-level
+   C++ and Java libraries and a Protobuf protocol definition; and
+3. Usable both through APIs like JDBC and ODBC thanks to this software
+   grant, as well as directly (or via [ADBC][adbc]) for applications
+   that want columnar data.
+
+## Getting Involved
+
+The JDBC driver was merged for the Arrow 10.0.0 release, and the
+[source code][impl] can be found in the Arrow repository.  Once Arrow
+10.0.0 is released (which is planned for October/November 2022),
```

Comment:
Revise this now that the RC is happening? And/or consider holding to publish with the release announcement--"you can check it out now" is a better call-to-action than "someday soon it will be released"